esbuild.config.mjs

```diff
@@ -160,6 +161,7 @@ esbuild.build({
   ],
   format: 'cjs',
   logLevel: "info",
+  minify: prod ? true : false,    
```

Comment:
Is this a change in behaviour that is not mentioned in the description and title of the PR?

Reply:
What I mean is, for my own education:

1. Did it used to not minify in prod and now it does, in which case it's an observable change in the release?
2. Or did it used to always minify and now it does not in dev, in which case it's just a nice convenience for developers?

Reply:
Conclusion from discussion: It was 1, it's now minified in releases, which will speed up vault load time.